README.md

```diff
@@ -176,7 +176,11 @@ client.notify("log", { message: "Hello, World!" });
 Just like `JSONRPCServer`, you can inject custom params to `JSONRPCClient` too:
 
 ```javascript
-const client = new JSONRPCClient(
+class AuthToken {
+  public token: string;
+}
```

Comment:
Oh thanks for the catch! I agree this is less confusing. One request is to use `interface` because I think that aligns with more common setups:
```suggestion
interface AuthToken {
  token: string;
}
```